src/screens/AuthLoadingScreen/styles.js

```diff
@@ -0,0 +1,12 @@
+import { StyleSheet } from 'react-native'
+
+const styles = StyleSheet.create({
+  mainContainer: {
+    flex: 1,
+    justifyContent: "center",
```

Comment:
```suggestion
    justifyContent: 'center',
```
A linter should fix this.